### PR TITLE
Make it easier for the next person to edit this site

### PR DESCRIPTION
**Pull request by pokle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
 Website for Compose Conference. Please visit us at [www.composeconference.org](http://www.composeconference.org)...
+
+
+### Contributions
+- Fork this repo
+- Start in various terminals:
+    - `bundle && bundle exec jekyll build --watch`
+    - `cd _site && python3 -m http.server 8000`
```
Comment (sordina) [inline README.md]: Shouldn't need to do this if you have jekyll...

Comment (sordina) [reply README.md]: `bundle exec jekyll serve --watch`

Comment (pokle) [reply README.md]: Nice! I didn’t know that

Comment (sordina) [reply README.md]: Feel free to amend the comment and push now that you have access :)

Comment (pokle) [reply README.md]: Oh, I'm not allowed to approve my own PRs. I think you'll have to help me here.

**Pull request merged**